apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/for-tup-cms/components/c-footer.css

```diff
@@ -174,6 +174,7 @@ html:not(#page-portal) .c-footer {
     border-width: var(--global-border-width--thick) 0 0 0;
     border-style: solid;
     border-color: var(--global-color-secondary--light);
+    padding-block: unset;
```

Comment:
This style is added within a block of CSS with a comment that says the code is—

```
/* To add a full-width border above CMS footer */
```

—but this fix is unrelated to full-width border.

Please find an alternate way to add the solution or arrange comments for this block.

Reply:
Ooh, the bug is on Portal too!

![Screenshot 2023-11-17 at 4 13 11 PM](https://github.com/TACC/tup-ui/assets/62723358/f1356b60-acad-4b25-a5d2-57297d691ded)

So maybe this fix can be in a simpler selector.

Reply:
I will fix portal padding manually with inline styles on the "bad" markup.

And you fixed the commenting already.